Cargo.toml

```diff
@@ -23,22 +23,25 @@ exclude = [
 all-features = true
 
 [features]
-default = ["rust_1_30", "backtraces"]
+default = ["rust_1_30", "backtraces", "std"]
+
+# Uses the `std::error::Error` trait instead of its own
+std = []
 
 # Adds the backtrace type
-backtraces = ["snafu-derive/backtraces", "backtrace"]
+backtraces = ["std", "snafu-derive/backtraces", "backtrace"]
```

Comment:
The backtrace crate has a `std` flag; why do we need to enforce it?

Reply:
When you want the backtrace crate, you have std and thus the std feature should be active such that you get the std::error::Error implementations and not the ones for the core Error trait we introduce here.

Reply:
Oh wait, you meant it works with no_std? I guess I'll try to get that working.

Reply:
Alright, I implemented the backtrace support. I was not quite able to test it properly though.

Reply:
Nvm, this doesn't build at all. You can't capture any backtraces with no_std, so I'd argue that we can't really support them until that's a thing in the backtrace crate.

Reply:
Sounds good, I guess that's just a placeholder functionality in their code.